src/test/java/autoracing/CarTest.java

```diff
@@ -0,0 +1,77 @@
+package autoracing;
+
+import org.assertj.core.api.Condition;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class CarTest {
+    private RacingRule rule;
+
+    @BeforeAll
+    public void init() {
+        this.rule = new RandomRacingRule(10, 4);
+    }
+
+    @Test
+    public void newCar() {
+        Car car = new Car(rule);
+        Condition<Car> init = new Condition<>(c -> c.getLocation(0) == Location.STARTING_LINE, "init");
+        assertThat(car).is(init);
```

Comment:
assertThat().is()를 사용하신 특별한 이유가 있을까요???
```suggestion
        assertThat(car.getLoction(0)).isEqualTo(Location.STARTING_LINE);
```
로도 충분할 것 같아서요:)